virtcontainers/api.go

```diff
@@ -653,3 +653,40 @@ func StatsContainer(sandboxID, containerID string) (ContainerStats, error) {
 
 	return s.StatsContainer(containerID)
 }
+
+func togglePauseContainer(sandboxID, containerID string, pause bool) error {
+	if sandboxID == "" {
+		return errNeedSandboxID
+	}
+
+	if containerID == "" {
+		return errNeedContainerID
+	}
+
+	lockFile, err := rLockSandbox(sandboxID)
+	if err != nil {
+		return err
+	}
+	defer unlockSandbox(lockFile)
+
+	s, err := fetchSandbox(sandboxID)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Get the container pointer from here, this way, you don't need to define sandbox functions for this. Let's avoid useless indirections.
This way, only
```go
func (c *Container) pause() error {}
func (c *Container) resume() error {}
```
are needed.